test/test_torch.py

```diff
@@ -2881,6 +2881,17 @@ def test_deepcopy_scalar(self, device, dtype):
         self.assertEqual(a.size(), deepcopy(a).size())
         self.assertEqual(a, deepcopy(a))
 
+    @dtypes(torch.float32)
```

Comment:
Note that if you just want a single test, you can just more it to `TestTorch` above and it will be called with no special argument and you can specify the type there directly.

Reply:
moved to TestTorch